### PR TITLE
LRAC-8997 | master

### PR DESCRIPTION
**Pull request by Tim-Cao**
https://issues.liferay.com/browse/LRAC-8997

This is only for master.

@alexcandidos

### CODE REVIEW

**Comment by Tim-Cao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualsAttributeBreakdown.testcase [hunk 1]
```diff
@@ -120,6 +120,78 @@ definition {
...
+
+		task ("Select the country in Breakdown by field") {
+			ACIndividualsDashboard.selectAttribute(
+				attribute = "country",
+				fieldLabel = "Breakdown By");
+		}
+
+		task ("View the individual in chart") {
+			ACIndividualsDashboard.viewBreakdownRecharts(
+				fillColor = "#4B9BFF",
```
Comment (christianstokes) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualsAttributeBreakdown.testcase:190]: Everything so far looks great in this test, good job!

@alexcandidos @Tim-Cao curious about asserting the chart color, is this necessary?

**Comment by alexcandidos** [comment]
ci:forward

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualsAttributeBreakdown.testcase (hunk 1 above)
Comment (Tim-Cao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualsAttributeBreakdown.testcase:190]: Good question. It's up to you. Feel free to let me know.

**Pull request closed without merging**